LICENSE

```diff
@@ -203,6 +203,12 @@
 
 APACHE NIFI - MINIFI SUBCOMPONENTS:
 
+The Apache NiFi - MiNiFi project contains subcomponents licensed under the Apache License, Version 2.0:
```

Comment:
I know it's just moved, but my gut feeling says that we should call the project MiNiFi C++ here as well to distinguish cpp and java versions. 

@apiri what's your opinion?

Reply:
yeah, that seems fair.  it was originally done as such from the standpoint that it was under a common effort/subproject but for licensing clarity, clearly differentiating will be helpful

Reply:
Agreed, will update all instances.